src/structure/pure/slide.js

```diff
@@ -182,6 +182,15 @@ design.basicdesign.addCreator('slide', {
     }
     transformSlider(ui.value,ui.handle);
   },
+  sliderVisible:function(event,page_id)
+  {
+    $('.ui-slider-handle.untransformed', '#'+page_id).each(function(i) {
+      $(this).removeClass('untransformed');
+      var actor = $(this).parent();
+      var val = actor.slider("value");
+      transformSlider(val,this);
+    });
+  }
   
```

Comment:
Whats this about? At least it doesn't seem to belong to the comma cleanup PR, right?